Makefile

```diff
@@ -85,6 +85,14 @@ manifests:
 	@$(WARN) Deprecated. Please run make generate instead.
 
 CRD_DIR = cluster/crds
+CRD_PATCH_DIR = cluster/crd-patches
+
+# See patch files for details.
+crds.patch: $(KUBECTL)
+	@$(INFO) patching generated CRDs
+	@$(KUBECTL) patch --local --type=json -f $(CRD_DIR)/pkg.crossplane.io_deploymentruntimeconfigs.yaml --patch-file $(CRD_PATCH_DIR)/pkg.crossplane.io_deploymentruntimeconfigs.yaml -o yaml > /tmp/pkg.crossplane.io_deploymentruntimeconfigs.yaml
```

Comment:
wasn't aware there is a local json patch mode in kubectl. Kind of odd feature :)

Reply:
Nor was I. 🙂 I copied this pattern from some other project that worked around a similar issue.